waiter/src/waiter/cookie_support.clj

```diff
@@ -59,10 +59,10 @@
 
 (defn add-encoded-cookie
   "Inserts the provided name-value pair as a Set-Cookie header in the response"
-  [response password name value age-in-days]
+  [response password name value age-in-days & [age-in-seconds]]
```

Comment:
We should have a single argument for the age.

Reply:
done